packages/next/src/migrations/update-9-2-0/create-next-config.ts

```diff
@@ -0,0 +1,44 @@
+import { Rule, Tree } from '@angular-devkit/schematics';
+import { readWorkspace } from '@nrwl/workspace';
+import { CONFIG_FILE } from 'next/dist/next-server/lib/constants';
+import { join } from 'path';
+
+const defaultConfig = `const withSass = require('@zeit/next-sass');
+const withLess = require('@zeit/next-less');
+const withStylus = require('@zeit/next-stylus');
+const withCSS = require('@zeit/next-css');
+
+/*
+ * This file was generated by Nx 9.2.0 to allow greater control over configuration
+ * for Next apps. To optimize your configuration please update the following export
+ * with only the style plugin you are using.
+ *
+ * e.g. module.exports = withCSS({ cssModules: false });
+ */
+module.exports = withStylus(withLess(withSass(withCSS({
+  // Set this to true if you use CSS modules.
+  // See: https://github.com/css-modules/css-modules
+  cssModules: false
+}))));
+`;
+
+export default function update(): Rule {
+  return (host: Tree) => {
+    const workspaceJson = readWorkspace(host);
+    const nextProjects = Object.keys(workspaceJson.projects)
+      .map(name => {
+        const p = workspaceJson.projects[name];
+        const buildBuilder =
+          p.architect && p.architect.build && p.architect.build.builder;
+        return buildBuilder === '@nrwl/next:build' ? p : null;
+      })
+      .filter(Boolean);
```

Comment:
How about optional chaining? 😁 

(might have to update prettier to get the formatting to play nice though)

```suggestion
    const nextProjects = Object.values<any>(workspaceJson.projects)
      .filter(p => p.architect?.build?.builder === '@nrwl/next:build');
```

You could, of course, do away with the `nextProjects` const entirely and just chain the above onto this `forEach`...

Reply:
Yeah I thought about it.. maybe it's time to update prettier.